### ISSUE TITLE
Pre-work Checklist: Developer: Daniel Perez

### ISSUE THREAD

**Issue by d-perez-8** [hackforla/website#3388]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by d-perez-8** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

This has taken about 8 hours to reach this point.

**Comment by HackforLABot** [comment]
## Activity Log: d-perez-8
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- d-perez-8 opened: https://github.com/hackforla/website/issues/3388 at 07/18/2022, 07:46 PM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3388 at 07/22/2022, 02:07 PM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2815#issuecomment-1046316414 at 07/25/2022, 10:34 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2815#issuecomment-1194423423 at 07/25/2022, 11:02 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3388#issuecomment-1194483292 at 07/25/2022, 11:55 AM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3427 at 07/25/2022, 12:29 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3427#issuecomment-1194544105 at 07/25/2022, 12:47 PM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3427#event-7068760501 at 07/26/2022, 04:46 PM PDT
- d-perez-8 issue closed by PR 3427: https://github.com/hackforla/website/issues/2815#event-7068760589 at 07/26/2022, 04:46 PM PDT
- d-perez-8 closed as completed: https://github.com/hackforla/website/issues/3388#event-7069295977 at 07/26/2022, 07:40 PM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3102#issuecomment-1116852709 at 07/28/2022, 08:54 AM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3438 at 07/28/2022, 11:28 AM PDT
- d-perez-8 closed: https://github.com/hackforla/website/pull/3438#event-7084003307 at 07/28/2022, 11:33 AM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3441 at 07/28/2022, 07:33 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3441#issuecomment-1198826952 at 07/28/2022, 07:52 PM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3441#event-7103815628 at 08/01/2022, 11:20 AM PDT
- d-perez-8 issue closed by PR 3441: https://github.com/hackforla/website/issues/3102#event-7103815715 at 08/01/2022, 11:20 AM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3339#issuecomment-1178240374 at 08/08/2022, 02:35 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3339#issuecomment-1208639456 at 08/08/2022, 02:38 PM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3454 at 08/08/2022, 04:54 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3454#issuecomment-1210007479 at 08/09/2022, 05:03 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3454#issuecomment-1210065337 at 08/09/2022, 07:02 PM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3454#event-7168558040 at 08/10/2022, 08:57 PM PDT
- d-perez-8 issue closed by PR 3454: https://github.com/hackforla/website/issues/3339#event-7168558083 at 08/10/2022, 08:57 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3463#issuecomment-1212686663 at 08/11/2022, 08:10 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3463#pullrequestreview-1070558909 at 08/11/2022, 08:27 PM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3336#issuecomment-1177001753 at 08/12/2022, 08:37 AM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3466 at 08/12/2022, 09:24 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3336#issuecomment-1213299843 at 08/12/2022, 09:26 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3466#issuecomment-1213638432 at 08/12/2022, 07:51 PM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3466#event-7182841602 at 08/12/2022, 10:59 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3466#issuecomment-1214168891 at 08/13/2022, 07:32 AM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2198#issuecomment-907765914 at 08/16/2022, 07:59 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2198#issuecomment-1217405355 at 08/16/2022, 08:06 PM PDT
- d-perez-8 closed by PR 3466: https://github.com/hackforla/website/issues/3336#event-7201906248 at 08/16/2022, 08:13 PM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3471 at 08/16/2022, 10:30 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3471#issuecomment-1218286271 at 08/17/2022, 10:12 AM PDT
- d-perez-8 issue closed by PR 3471: https://github.com/hackforla/website/issues/2198#event-7228040529 at 08/21/2022, 11:31 AM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3471#event-7228040505 at 08/21/2022, 11:31 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3490#issuecomment-1229228876 at 08/27/2022, 10:07 AM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3490#pullrequestreview-1087704606 at 08/27/2022, 10:38 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3490#issuecomment-1229237944 at 08/27/2022, 11:04 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3490#issuecomment-1229243060 at 08/27/2022, 11:36 AM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3490#pullrequestreview-1087897477 at 08/28/2022, 11:29 AM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2199#issuecomment-907766320 at 08/29/2022, 10:26 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2199#issuecomment-1230618486 at 08/29/2022, 10:29 AM PDT
- d-perez-8 opened: https://github.com/hackforla/website/pull/3497 at 08/29/2022, 10:46 AM PDT
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3497#event-7282462531 at 08/29/2022, 03:03 PM PDT
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2286#issuecomment-922516418 at 09/06/2022, 01:49 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1238636738 at 09/06/2022, 01:51 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1238655483 at 09/06/2022, 02:13 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3534#issuecomment-1245581150 at 09/13/2022, 08:29 AM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3534#pullrequestreview-1106050362 at 09/13/2022, 08:45 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1250348436 at 09/18/2022, 10:03 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3585#issuecomment-1258829426 at 09/26/2022, 05:46 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3585#pullrequestreview-1121081231 at 09/26/2022, 05:55 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3580#issuecomment-1258834855 at 09/26/2022, 05:57 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3580#pullrequestreview-1121090058 at 09/26/2022, 06:13 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1267310366 at 10/04/2022, 10:12 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3599#issuecomment-1267526197 at 10/04/2022, 01:14 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3599#pullrequestreview-1130678586 at 10/04/2022, 03:23 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1272364476 at 10/08/2022, 10:33 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3600#issuecomment-1272587782 at 10/09/2022, 10:15 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3601#issuecomment-1273599122 at 10/10/2022, 10:04 AM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3601#pullrequestreview-1136341154 at 10/10/2022, 10:24 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1276577690 at 10/12/2022, 11:34 AM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3620#issuecomment-1279540360 at 10/14/2022, 03:30 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3620#pullrequestreview-1143082885 at 10/14/2022, 03:46 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1281620730 at 10/17/2022, 04:21 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3629#issuecomment-1281621743 at 10/17/2022, 04:22 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3629#pullrequestreview-1144920966 at 10/17/2022, 04:36 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3633#issuecomment-1281636086 at 10/17/2022, 04:44 PM PDT
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3633#pullrequestreview-1146737001 at 10/18/2022, 06:44 PM PDT
- d-perez-8 unassigned: https://github.com/hackforla/website/issues/2286#issuecomment-1286080470 at 10/23/2022, 10:52 AM PDT
- d-perez-8 unassigned: https://github.com/hackforla/website/issues/2198#event-7228040529 at 11/06/2022, 11:15 AM PST
- d-perez-8 unassigned: https://github.com/hackforla/website/issues/2199#event-7282462601 at 11/06/2022, 11:15 AM PST
- d-perez-8 issue closed by PR 3471: https://github.com/hackforla/website/issues/2198#event-7747786281 at 11/06/2022, 11:24 AM PST
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2198#issuecomment-1304874482 at 11/08/2022, 11:57 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3718#issuecomment-1314634006 at 11/14/2022, 05:30 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3718#pullrequestreview-1180081974 at 11/14/2022, 05:46 PM PST
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3761#issuecomment-1336128568 at 01/19/2023, 04:36 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3761#issuecomment-1397786950 at 01/19/2023, 04:37 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/pull/3845 at 01/19/2023, 05:57 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3852 at 01/22/2023, 11:31 AM PST
- d-perez-8 assigned: https://github.com/hackforla/website/issues/2616#issuecomment-1272583281 at 01/22/2023, 11:37 AM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3853 at 01/22/2023, 12:00 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3854 at 01/22/2023, 12:03 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3855 at 01/22/2023, 12:05 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3856 at 01/22/2023, 12:11 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3857 at 01/22/2023, 12:13 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/issues/2616#issuecomment-1399597413 at 01/22/2023, 12:21 PM PST
- d-perez-8 unassigned: https://github.com/hackforla/website/issues/2616#issuecomment-1399614108 at 01/22/2023, 01:33 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3847#issuecomment-1401126521 at 01/23/2023, 03:05 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3847#pullrequestreview-1266600511 at 01/23/2023, 03:42 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3850#issuecomment-1401158710 at 01/23/2023, 03:44 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3850#pullrequestreview-1266613042 at 01/23/2023, 04:00 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3845#issuecomment-1407283446 at 01/27/2023, 08:21 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3883#issuecomment-1407733079 at 01/29/2023, 10:14 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3845#issuecomment-1407739131 at 01/29/2023, 10:40 AM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3883#pullrequestreview-1276123558 at 01/30/2023, 02:19 PM PST
- d-perez-8 opened: https://github.com/hackforla/website/issues/3903 at 01/30/2023, 03:22 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3896#issuecomment-1411143510 at 01/31/2023, 02:16 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3883#pullrequestreview-1278122507 at 01/31/2023, 02:32 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3896#pullrequestreview-1278147870 at 01/31/2023, 02:54 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3887#issuecomment-1411199005 at 01/31/2023, 03:09 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/3887#pullrequestreview-1278196868 at 01/31/2023, 03:50 PM PST
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/3845#event-8405553638 at 01/31/2023, 06:15 PM PST
- d-perez-8 issue closed by PR 3845: https://github.com/hackforla/website/issues/3761#event-8405553763 at 01/31/2023, 06:15 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3887#issuecomment-1416879632 at 02/04/2023, 03:59 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/3887#issuecomment-1421579595 at 02/07/2023, 02:52 PM PST
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3931#issuecomment-1427107968 at 02/21/2023, 09:38 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3931#issuecomment-1438874061 at 02/21/2023, 09:45 AM PST
- d-perez-8 opened: https://github.com/hackforla/website/pull/4020 at 02/21/2023, 10:38 AM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4019#pullrequestreview-1308018388 at 02/21/2023, 11:05 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4020#issuecomment-1440905087 at 02/22/2023, 02:31 PM PST
- d-perez-8 closed: https://github.com/hackforla/website/pull/4020#event-8603742243 at 02/24/2023, 08:38 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4020#issuecomment-1444007126 at 02/24/2023, 08:52 AM PST
- d-perez-8 opened: https://github.com/hackforla/website/pull/4034 at 02/24/2023, 08:55 AM PST
- d-perez-8 PR was merged: https://github.com/hackforla/website/pull/4034#event-8610114438 at 02/26/2023, 12:16 PM PST
- d-perez-8 issue closed by PR 4034: https://github.com/hackforla/website/issues/3931#event-8610114477 at 02/26/2023, 12:16 PM PST
- d-perez-8 assigned: https://github.com/hackforla/website/issues/3805#issuecomment-1444584350 at 03/01/2023, 02:56 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1450978723 at 03/01/2023, 02:57 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4080#issuecomment-1451156157 at 03/01/2023, 05:54 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4080#issuecomment-1451196763 at 03/01/2023, 06:54 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4091#issuecomment-1452415011 at 03/02/2023, 11:07 AM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4091#pullrequestreview-1322476139 at 03/02/2023, 11:22 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4086#issuecomment-1452432108 at 03/02/2023, 11:23 AM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4086#pullrequestreview-1322496597 at 03/02/2023, 11:37 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4093#issuecomment-1454216535 at 03/03/2023, 02:33 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4093#pullrequestreview-1324701863 at 03/03/2023, 02:46 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4093#pullrequestreview-1324789954 at 03/03/2023, 04:36 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4094#issuecomment-1454930123 at 03/04/2023, 04:06 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4094#pullrequestreview-1325138692 at 03/04/2023, 04:15 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4106#issuecomment-1457316842 at 03/06/2023, 05:14 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4106#pullrequestreview-1327509133 at 03/06/2023, 05:51 PM PST
- d-perez-8 commented on: https://github.com/hackforla/website/pull/4107#issuecomment-1457372859 at 03/06/2023, 05:59 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4107#pullrequestreview-1327530812 at 03/06/2023, 06:15 PM PST
- d-perez-8 submitted review: https://github.com/hackforla/website/pull/4106#pullrequestreview-1331193825 at 03/08/2023, 10:12 AM PST
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1477079736 at 03/20/2023, 04:22 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1487845034 at 03/28/2023, 07:06 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1487889927 at 03/28/2023, 08:19 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1487897003 at 03/28/2023, 08:32 PM PDT
- d-perez-8 commented on: https://github.com/hackforla/website/issues/3805#issuecomment-1522449302 at 04/25/2023, 02:32 PM PDT
- d-perez-8 unassigned: https://github.com/hackforla/website/issues/3805#issuecomment-1545289300 at 05/14/2023, 07:46 PM PDT
